### DISCUSSION TITLE
posts/pulumi-azure-backend

### DISCUSSION THREAD

**Discussion by giscus** [TechWatching/techwatching.dev#26] [Announcements]
# posts/pulumi-azure-backend

https://www.techwatching.dev/posts/pulumi-azure-backend

**Comment by antonbabenko** [comment]
`"private"` key in Terraform state file does not contain anything secret. It is a bad name for the JSON key, but here is what your tfstate has:

```
$ echo "eyJlMmJmYjczMC1lY2FhLTExZTYtOGY4OC0zNDM2M2JjN2M0YzAiOnsiZGVsZXRlIjoxODAwMDAwMDAwMDAsInJlYWQiOjkwMDAwMDAwMDAwMCwidXBkYXRlIjozMDAwMDAwMDAwMDB9fQ==" | base64 -d
{"e2bfb730-ecaa-11e6-8f88-34363bc7c4c0":{"delete":180000000000,"read":900000000000,"update":300000000000}}
```

**Reply by TechWatching** [reply]
@antonbabenko Yes you are totally right, I did not highlight the correct part of the terraform state file. I have replaced the screenshot to show the secret I was talking about. Thanks for noticing.
